source/fonts.css

```diff
@@ -1,76 +1,54 @@
-@font-face {
-	font-family: 'Proxima Nova';
-	src: url('./fonts/');
-	src: local('Proxima Nova'), local('ProximaNovaT-Thin'),
-		url('./fonts/?#iefix') format('embedded-opentype'),
-		url('./fonts/ProximaNovaT-Thin.woff') format('woff'),
-		url('./fonts/ProximaNovaT-Thin.ttf') format('truetype');
-	font-weight: 100;
-	font-style: normal;
-}
```

Comment:
Is there no thin equivalent for creato?

Reply:
There is. Adding it now. Forgot to include it in webfont generator. Updating it now.